styleguide/source/_patterns/00-atoms/04-images/09-landscape-175x105.twig

```diff
@@ -0,0 +1 @@
+<img {% if class %}class='{{ class }}'{% endif %} src="{{ src|default('https://ipsumimage.appspot.com/175x105') }}" alt="This is an image.">
```

Comment:
Added `Remove the image atoms from the atoms directory and replace with a landscape atom and a portrait atom placeholder.` to https://issues.ama-assn.org/browse/EWL-3484 as well.